src/lib/Config.ts

```diff
@@ -0,0 +1,120 @@
+export default class Config {
+
+    // Signal if the app should be started in offline mode
+    public static offline = navigator.onLine !== undefined && !navigator.onLine;
+
+    // When packaged as distributed offline app, subdomains are folder names instead
+    public static offlinePackaged = false;
+
+    /* Public methods */
+
+    public static origin(subdomain: string) {
+        return Config._origin(subdomain);
+    }
+
+    public static src(subdomain: string) {
+        return Config._origin(subdomain, true);
+    }
+
+    public static get tld() {
+        const tld = Config._myOrigin.split('.');
+        return [tld[tld.length - 2], tld[tld.length - 1]].join('.');
```

Comment:
Technically, the TLD is only the last part of the origin, e.g., `.com` or `.de`.
But besides the naming, there is another problem when it comes to TLDs like `.co.uk`.
`test.co.uk` and `evil.co.uk` would yield the same result here!

Reply:
As far as I can see, this will only be invoked from our domains (currently under the `.com` TLD).
So, I would at least add a comment describing the problem here.